libs/openFrameworks/graphics/ofPolyline.inl

```diff
@@ -93,6 +93,19 @@ void ofPolyline_<T>::insertVertex(float x, float y, float z, int index) {
 }
 
 
+//----------------------------------------------------------
+template<class T>
+void ofPolyline_<T>::removeVertex(int index) {
+	if(index >= points.size()){
```

Comment:
this kind of bounds check only happens in c++ in debug mode, not sure we should have it

Reply:
I just copied this pattern from ofMesh.  I'd prefer to remove it.  Should we keep it for consistency?

Reply:
oh, ok, letr's leave it then although it's weird